### ISSUE TITLE
Can't Install App

### ISSUE THREAD

**Issue by aiartsev** [HHK1/PryntTrimmerView#63]
After integrating this framework using Carthage and trying to run on a device, I get the following message:

Unable to install "AppName"
Domain: com.apple.dt.MobileDeviceErrorDomain
Code: -402653103
--
Could not inspect the application package.
Domain: com.apple.dt.MobileDeviceErrorDomain
Code: -402653103
User Info: {
    DVTRadarComponentKey = 282703;
    MobileDeviceErrorCode = "(0xE8000051)";
    "com.apple.dtdevicekit.stacktrace" = (
	0   DTDeviceKitBase                     0x000000011e6a781a DTDKCreateNSErrorFromAMDErrorCode + 233
	1   DTDeviceKitBase                     0x000000011e6e8f70 __90-[DTDKMobileDeviceToken installApplicationBundleAtPath:withOptions:andError:withCallback:]_block_invoke + 155
	2   DVTFoundation                       0x00000001043d7155 DVTInvokeWithStrongOwnership + 73
	3   DTDeviceKitBase                     0x000000011e6e8ca8 -[DTDKMobileDeviceToken installApplicationBundleAtPath:withOptions:andError:withCallback:] + 1654
	4   IDEiOSSupportCore                   0x000000011e55fe91 __118-[DVTiOSDevice(DVTiPhoneApplicationInstallation) processAppInstallSet:appUninstallSet:installOptions:completionBlock:]_block_invoke.352 + 4165
	5   DVTFoundation                       0x000000010450a7f4 __DVT_CALLING_CLIENT_BLOCK__ + 7
	6   DVTFoundation                       0x000000010450c436 __DVTDispatchAsync_block_invoke + 1194
	7   libdispatch.dylib                   0x00007fff6b7556c4 _dispatch_call_block_and_release + 12
	8   libdispatch.dylib                   0x00007fff6b756658 _dispatch_client_callout + 8
	9   libdispatch.dylib                   0x00007fff6b75bc44 _dispatch_lane_serial_drain + 597
	10  libdispatch.dylib                   0x00007fff6b75c5d6 _dispatch_lane_invoke + 363
	11  libdispatch.dylib                   0x00007fff6b765c09 _dispatch_workloop_worker_thread + 596
	12  libsystem_pthread.dylib             0x00007fff6b9b0a3d _pthread_wqthread + 290
	13  libsystem_pthread.dylib             0x00007fff6b9afb77 start_wqthread + 15
);
}
--

System Information

macOS Version 10.15.5 (Build 19F101)
Xcode 11.5 (16139)
iOS 13.5.1

**Comment by HHK1** [comment]
Yup, sorry about that, I'm not familiar with Carthage and must have broken the pre-built binary in the latest release. 
I'll look at it in the morning, in the meantime, can you target the previous version ?

**Comment by aiartsev** [comment]
Yup, no worries man! Let me know when it's okay to update

**Comment by NikKovIos** [comment]
@HHK1 In order the SPM is strongly recommended by apple, what if we deprecate carthage and focus on SPM and cocoapods only?
